lib/player.rb

```diff
@@ -0,0 +1,81 @@
+require_relative 'scoring'
+require_relative 'tilebag'
+
+module Scrabble
+  class Player
+    attr_reader :name, :played_words, :tiles
+    def initialize(name)
+      raise ArgumentError.new("Name must be a string") if name.class != String
+      @name = name
+      @played_words = []
+      @total_score = 0
+      @tiles = []
+    end
+
+    def total_score
+      @played_words.each do |word|
+        @total_score += Scrabble::Scoring.score(word)
+      end
+      return @total_score
+    end
+
+    def play(word)
+      raise ArgumentError.new("Word must be a string") if word.class != String
+      raise ArgumentError.new("Word must be 7 letters or less") if word.length > 7
+      # We abandoned trying to verify that a player has the needed tiles because
+      # it looks like we would have to adjust a bunch of our tests.
+      # Question: is it normal/acceptable to go back and change tests?
+      # Or is that indicative of serious design problems?
```

Comment:
As a rule: It's okay to adjust tests retroactively! Does this indicate a design problem? Totally depends on the context!
I'm a little curious about why you'd have to adjust tests to make that new feature work.